lib/abci/handlers/deliverTxHandlerFactory.js

```diff
@@ -48,26 +57,32 @@ function deliverTxHandlerFactory(dpp, driveUpdateStateClient, blockchainState) {
       throw e;
     }
 
-    const applyStateTransitionRequest = new ApplyStateTransitionRequest();
+    const stType = stateTransition.getType();
+    if (stType === stateTransitionTypes.IDENTITY_CREATE) {
+      const identityModel = new IdentityModel();
```

Comment:
I've added proper export with validations and stuff, so there's an `dpp.identity.create()` method now

Reply:
There's also a `dpp.identity.validate()` method in case if you want to validate the model